lib/lpc55-rot-startup/src/lib.rs

```diff
@@ -290,3 +323,5 @@ extern "C" fn nuke_stack() {
         )
     }
 }
+
+lpc55_hashcrypt::dynamic_hashcrypt!();
```

Comment:
Do we we need this or can we go to the non-procmacro version

Reply:
Already removed